application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -45,7 +46,11 @@ public void onMessageReceived(MessageReceivedEvent event) {
         }
 
         if (messageHasNoMediaAttached(message)) {
-            message.delete().flatMap(any -> dmUser(message)).queue();
+            message.delete().flatMap(any -> dmUser(message)).queue(any -> {
+            }, err -> {
+                pingUserAobutMedia(message)
```

Comment:
* typo. `About`, not `Aobut`
* also kinda a bad name, "about media" - what about? what happened? and its also too specific with the "ping" - can quickly get out of date if someone changes its internals
* better would be to go with the current pattern in this class, where it says `dmUser` and now u can instead use `pingUserInChannel`. also makes clear why its the alternative

Reply:
I renamed and uniied the function can u just go through them the newer set?